### PR TITLE
Resolve PIL transfers

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: lib/resolvers/pil.js [hunk 1]
```diff
@@ -3,7 +3,26 @@ const { pick } = require('lodash');
 const { generateLicenceNumber } = require('../utils');
 
 module.exports = ({ models }) => ({ action, data, id }, transaction) => {
-  const { PIL } = models;
+  const { PIL, PilTransfer } = models;
+
+  if (action === 'transfer') {
+    return PIL.query(transaction).findById(id)
+      .then(pil => pil.$query(transaction).patchAndFetch({
```
Comment (lennym) [inline lib/resolvers/pil.js]: What's the difference between `.query()` and `.$query()`?

File: lib/resolvers/pil.js [hunk 2]
```diff
@@ -3,7 +3,26 @@ const { pick } = require('lodash');
 const { generateLicenceNumber } = require('../utils');
 
 module.exports = ({ models }) => ({ action, data, id }, transaction) => {
-  const { PIL } = models;
+  const { PIL, PilTransfer } = models;
```
Comment (lennym) [inline lib/resolvers/pil.js]: I should have beefed about `PIL` vs `Pil` in your other PR.

The 💩 is out of the 🐴 now though.

File: lib/resolvers/pil.js (hunk 1 above)
Comment (wheelsandcogs) [reply lib/resolvers/pil.js]: `.query()` is on the Model class, `.$query()` is on an instance of the Model so it knows which record to update.

File: lib/resolvers/pil.js (hunk 2 above)
Comment (wheelsandcogs) [reply lib/resolvers/pil.js]: I considered both, but `PIL` already causes us grief in some places due to the !camelCase . Also, we've also already got `AsruEstablishment`

**Pull request merged**